src/MoviesContainer/MoviesContainer.js

```diff
@@ -2,24 +2,48 @@ import React from 'react';
 import MovieCard from '../MovieCard/MovieCard';
 import './MoviesContainer.css';
 
-const MoviesContainer = ({movies}) => {
-    const movieCards = movies.map(movie => {
-        return (
-            <MovieCard 
-                id={movie.id}
-                key={movie.id}
-                image={movie.poster_path}
-                backdrop={movie.backdrop_path}
-                title={movie.title}
-                rating={movie.average_rating}
-            />
-        )
+const MoviesContainer = ({ movies, moviesByRating }) => {
+  if (moviesByRating[0] === "NaN") {
```

Comment:
I like how our if statements work. I think this is the best way to do this. At least at my experience so far, it works!

Reply:
I agree!